packages/rule-http-cache/src/rule.ts

```diff
@@ -89,16 +89,48 @@ export default class HttpCacheRule implements IRule {
             'video/webm'];
         /** The predefined patterns for file revving.*/
         const predefinedRevvingPatterns: Array<RegExp> = [
+            /** https://regex101.com/r/6VduJO/20/ */
+
+            /*
+             * E.g.: semver
+             * - https://example.com/assets/jquery-2.1.1.js
+             * - https://example.com/assets/jquery-2.1.1.min.js
+             * - https://example.com/assets/favicon.123.ico
+             *
+             * Live example: https://regex101.com/r/6VduJO/19/
+             */
+            /\/[^/]+[._-]v?\d+(\.\d+(\.\d+)?)?[^/]*\.\w+$/i,
+
+            /*
+             * E.g.: last modified timestamp
+             * - https://example.com/assets/js.script-1234567890.js
+             *
+             * Live example: https://regex101.com/r/6VduJO/8/
+             */
+            /\/[^/]+[._-](\d{10,})\.\w+$/i,
+
+            /*
+             * E.g.: file hash
+             * - https://example.com/assets/unicorn-d41d8cd98f.css
+             * - https://example.com/assets/app.9f61f58dd1cc3bb82182.bundle.js
+             *
+             * Live example: https://regex101.com/r/6VduJO/21/
+             *
+             * - https://example.com/assets/9f61f58dd1cc3bb82182.bundle.js
+             *
+             * Live example: https://regex101.com/r/6VduJO/22/
+             */
+            /\/[^/]+[._-]([0-9a-f]{10,})(\.bundle)?\.\w+$/i,
```

Comment:
Why are we using the limit of 10?

There are a lot of websites that don't use a hash that long.

Reply:
I'm trying to find stats for that number but can't seem to find anything conclusive. webpack's default is 20 IIRC, gulp-rev is 10. Don't know about other popular tools.